.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+version: 2
+
+jobs:
+  checkout:
+    working_directory: ~/openfisca-core
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - checkout
+
+      - save_cache:
+          key: v1-checkout-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - .
+
+  dependencies:
+    working_directory: ~/openfisca-core
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - restore_cache:
+          keys:
+            - v1-checkout-{{ .Environment.CIRCLE_SHA1 }}
+
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ .Environment.CIRCLE_SHA1 }}
+
+      - run:
+          name: Create virtualenv
```

Comment:
Why do we need to create a virtualenv to test one Core revision?

Reply:
Because we're passing dependencies from job to job: each job corresponds to an empty Docker image.